uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/LeaveRequest.php

```diff
@@ -375,4 +383,46 @@ private function skipBalanceChangeUpdate($params) {
 
     return ($leaveRequestDatesNotChanged || $toilDatesAndToAccrueNotChanged) && !$updateBalanceChange;
   }
+
+  /**
+   * Checks If the current user can cancel a TOIL request type that has past dates.
+   *
+   * @param array $params
+   *
+   * @return bool
+   */
+  private function canCanCancelTOILWithPastDates($params) {
+    return $this->leaveRequestRightsService->canCancelToilWithPastDates($params['contact_id'], $params['type_id']);
```

Comment:
This line is long, can you adjust the indentation like

```
return $this->leaveRequestRightsService->canCancelToilWithPastDates(
  $params['contact_id'], 
  $params['type_id']
);
```

Reply:
Ok